src/goto-symex/renaming_level.cpp

```diff
@@ -97,6 +136,12 @@ void symex_level1t::restore_from(const current_namest &other)
   }
 }
 
+unsigned symex_level2t::current_count(const irep_idt &identifier) const
```

Comment:
This name sounds too much like `std::map::count`, how about `latest_renaming_index`?

Reply:
Maybe `latest_index`? "renaming" is kind of implied when talking about `symex_level2t`